go/libkb/globals.go

```diff
@@ -30,6 +30,14 @@ import (
 
 type ShutdownHook func() error
 
+type LoginHook interface {
+	OnLogin()
```

Comment:
What about on startup?  Will gregor be available if you have a valid session token (from last run) but don't have an unlocked key?